paasta_tools/utils.py

```diff
@@ -827,10 +829,29 @@ def get_sensu_port(self):
     def get_dockercfg_location(self):
         """Get the location of the dockerfile, as a URI.
 
-        :returns: the URI speicfied, or file:///root/.dockercfg if not specified.
+        :returns: the URI specified, or file:///root/.dockercfg if not specified.
         """
         return self.get('dockercfg_location', DEFAULT_DOCKERCFG_LOCATION)
 
+    def get_synapse_port(self):
+        """Get the port that haproxy-synapse exposes its status on. Defaults to 3212.
+
+        :returns: the haproxy-synapse status port."""
+        return int(self.get('synapse_port', 3212))
+
+    def get_default_synapse_host(self):
+        """Get the default host we should interrogate for haproxy-synapse state.
+
+        :returns: A hostname that is running haproxy-synapse."""
+        return self.get('synapse_host', 'localhost')
+
+    def get_synapse_haproxy_url_format(self):
```

Comment:
I've made a bunch of changes so that this gets called properly.

Can you take a look again and make sure they look sane?